lib/widgets.php

```diff
@@ -3,7 +3,7 @@
  * Widget callback functions.
  *
  * @package Genesis_Connect_WooCommerce
- * @since 1.0.1
+ * @since 1.1.0
```

Comment:
Was this added in this last round of revisions? Or did this just get caught in a search/replace?

Reply:
@nathanrice I assumed the version would be `1.0.1` at one point. So I went back and updated `1.0.1` to `1.1.0`.